armi/utils/tests/test_utils.py

```diff
@@ -36,6 +36,9 @@ def test_parabola(self):
         self.assertEqual(b, 0.0)
         self.assertEqual(c, 1.0)
 
+        with self.assertRaises(Exception):
```

Comment:
probably should get more specific than `Exception`. I get a `LinAlgError: Singular matrix`, so would `assertRaises(numpy.linalg.LinAlgError)` be better? Meh, maybe not. We're really just trying to make sure it fails and probably shouldn't be too sensitive to the internals of numpy. Ok you can leave it.

Reply:
I didn't want to modify the actual code, which just has a "raise" statement with no specific exception/error type given.